core/fuzzer.cpp

```diff
@@ -132,6 +134,10 @@ p4::v1::TableEntry P4RuntimeFuzzer::produceTableEntry(
     const auto &matchFields = table.match_fields();
     for (auto i = 0; i < matchFields.size(); i++) {
         auto match = matchFields[i];
+        /// TODO: add this back once Flay supports OPTIONAL.
```

Comment:
Optional matches are a little strange. They can be omitted entirely or are an exact match. Let me see.